src/libs/Url.js

```diff
@@ -10,7 +10,51 @@ function addTrailingForwardSlash(url) {
     return url;
 }
 
+/**
+ * Parse href to URL object
+ * @param {String} href
+ * @returns {Object}
+ */
+function getURLObject(href) {
+    const urlRegex = new RegExp([
+        '^(https?:)//', // protocol
```

Comment:
I still feel like we need to update this regex as it doesn't parse URL's without http protocol

Reply:
can we use the regex from [ExpensiMark.js](https://github.com/Expensify/expensify-common/blob/dd66d931aeffe18cdcffab8e7d05af32f3ef7ad1/__tests__/ExpensiMark-HTML-test.js)

Reply:
You wanted to link this - https://github.com/Expensify/expensify-common/blob/dd66d931aeffe18cdcffab8e7d05af32f3ef7ad1/lib/ExpensiMark.js#L5?
If yes, sounds good 👍

Reply:
Yeah, that one. Thanks.

Reply:
@parasharrajat @MonilBhavsar Thanks for your review, but I'm afraid that we have no way to import `URL_WEBSITE_REGEX` from `ExpensiMark`. Should I duplicate `URL_WEBSITE_REGEX` and `TLD_REGEX` or fix on upstream.

Reply:
I would say fix on upstream. @parasharrajat what do you think?

Reply:
yeah, feel free to send a PR to expose the data from the e-common repo. https://github.com/Expensify/expensify-common